test/python/utils.py

```diff
@@ -237,3 +237,21 @@ def check_package_files(project, basedir):
 
     # check version file
     validate_version_file(project, distribution_dir)
+
+
+def run_command_and_get_output(cmd, cwd=None, env=None):
+    process = subprocess.Popen(
+        cmd,
+        env=env,
+        cwd=cwd,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE
+    )
+
+    stdout, _ = process.communicate()
+    stdout = stdout.decode('utf-8')
+
+    # This print is here to make running tests with -s flag more verbose
+    print(stdout)
```

Comment:
May be `log`?

Reply:
Why?

Reply:
I think that uncontrolled output into stdout is a mauvais

Reply:
This print is here only to prevent adding it on every `pytest -s` call (I use it to debug some things in `cartridge-cli` itself). I don't see any problems here, there is `cartridge-cli` output anywayю